src/test/java/CRTGearRatioUtilTestingStuff.java

```diff
@@ -0,0 +1,43 @@
+import frc.robot.utils.CRTGearRatioUtil;
+import org.junit.jupiter.api.Test;
+import java.util.Optional;
+
+public class CRTGearRatioUtilTestingStuff {
+  @Test
+  void test() {
+    assert checker(44, 42, 22);
+    assert checker(44, 26, 22);
+    assert !checker(3, 4, 22);
+    assert !checker(4,3,10);
+    assert checker(28,30,14);
+  }
+  /*Main gear ratio, secondary gear ratio, max rotations */
+  boolean checker(int mgr, int sgr, int mr){
```

Comment:
Fix function comment format. These should be `@param` and there should be a main function comment that says what this is doing. Returns false if the arguments are an invalid CRT setup

Reply:
Fixed.